ceph/rbd/workflows/rbd_mirror.py

```diff
@@ -161,6 +161,29 @@ def wait_for_status(rbd, cluster_name, **kw):
             raise Exception("Required status can not be attained")
 
 
+def wait_for_replay_complete(rbd, cluster_name, imagespec):
+    """Waits till image replay to complete in journal based mirroring.
+
+    Args:
+        imagespec: image specification.
+    """
+    log.info(f"Waiting for {imagespec} to complete replay")
+    while True:
+        time.sleep(30)
+        out = wait_for_status(
+            rbd=rbd,
+            imagespec=imagespec,
+            cluster_name=cluster_name,
+            description_pattern="entries",
+        )
+        out1 = out.split('entries_behind_primary":')
+        out2 = out1[1].split(",")
+        log.debug(f"entries_behind_primary : {out2[0]}")
+        if int(out2[0]) == 0:
```

Comment:
if `entries_behind_primary` never comes to 0 then it stucks in infinite loop, can we break this loop on some condition?

Reply:
That's how it has been always, and it will anyways timeout with the global timeout if the entries don't come down to 0. So I don't think this validation is required.